client/server/server.js

```diff
@@ -18,6 +18,17 @@ app.use(
     })
 )
 
+app.use(
+    '/api',
```

Comment:
Remove if unused
(we are using our clientToDBAPi, update Rom about it as well )